specs/phase0/p2p-interface.md

```diff
@@ -469,16 +475,18 @@ constituents individually as `response_chunk`s. For example, the
 Request, Response Content:
 ```
 (
-  head_fork_version: Bytes4
-  finalized_root: Bytes32
-  finalized_epoch: uint64
-  head_root: Bytes32
-  head_slot: uint64
```

Comment:
I feel @arnetheduck wanted to use simpler types instead of custom types in the first networking spec draft, not sure about what do you think about it now?

IMO it's okay to use custom types for presentation here.

Reply:
I think consistency is better, but don't feel super strongly about it

Reply:
fine with consistency across specs as argument - it wasn't a strong opinion - it also helps when grep:ing to have similar names and types across spec. :+1: